assets/src/dashboard/app/views/shared/bodyWrapper.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 Google LLC
```

Comment:
I moved the shared components into a folder named `shared`, but should I just call it `components` and it's location the file hierarchy would imply that it's components specifically for `views`?

Reply:
i like`shared` because it's implicit that these are shared views. I think you should leave this.